src/content/overview/developer-portal/clusters/details/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Cluster details
+linkTitle: Cluster details
+description: About the cluster details page in the Giant Swarm developer portal.
+weight: 20
+menu:
+  principal:
+    parent: overview-developer-portal-clusters
+    identifier: overview-developer-portal-clusters-details
+last_review_date: 2025-04-01
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I access cluster details in the developer portal?
+---
+
+![Cluster details page](./cluster-details.png)
+
+For each cluster, regardless whether workload or management cluster, you find a details page in the developer portal. The URI path of this page has the following schema:
+
+```nohighlight
+/clusters/{MC_NAME}/{MC_NAMESPACE}/{CLUSTER_NAME}
+```
+
+Here, `MC_NAME` is the name of the management cluster, `MC_NAMESPACE` is the namespace in the management cluster holding the cluster resources, and `CLUSTER_NAME` is the name of the cluster.
+
```

Comment:
I wonder here if we're overloading NAMESPACE references. Some of our documentation refers to this as `ORG_NAMESPACE` and others refers to it as simply `NAMESPACE` - would it be worth considering consolidating these around a single reference type?

Reply:
Makes sense. Changed to ORG_NAMESPACE.